spec/openapi-spec.yaml

```diff
@@ -0,0 +1,40 @@
+openapi: 3.1.1
+info:
+  title: Lance REST Catalog API
+  version: 0.0.1
+  description: REST interface for managing Lance dataset or LanceDB databases and tables
+
+paths:
+  /v1/databases:
+    post:
+      tags: [ Database ]
+      summary: Create new database
+      operationId: CreateDatabase
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateDatabaseRequest'
+      responses:
+        201:
+          description: Database created
+          headers:
+            Location:
+              description: URI of created database
```

Comment:
here you explicitly return a location, seems like location is a rather important concept. but why is that no a part of the create database request?

Reply:
After thinking, I think not returning anything is a better choice. WDYT?